lib/model.js

```diff
@@ -305,6 +305,10 @@ module.exports = class Model extends EventEmitter {
 
   url(options) {
     options = options || {};
+
+    // use this.options.url if it's been set
+    options.url = this.options.url || options.url;
```

Comment:
I think this should be the other way around. If someone calls `model.save({ url: 'http://example.com' })` then that should take precedent.